apps/documents/assets/ChapterList.jsx

```diff
@@ -0,0 +1,37 @@
+var React = require('react')
+var django = require('django')
+var ChapterListItem = require('./ChapterListItem')
+
+const ChapterList = React.createClass({
```

Comment:
I feel like the name `ChapterList` is a bit confusing. Maybe `ChapterNav` would be better. I am not convniced though.